cognite/neat/_session/_state.py

```diff
@@ -162,10 +161,10 @@ def lookup_containers(
         if missing := set(container_ids) - set(self._cdf_containers.keys()):
             cdf_container_ids = list(missing)
             found = client.data_modeling.containers.retrieve(cdf_container_ids).as_write()
-            if not_found := missing - set(found.as_ids()):
```

Comment:
Realized that a created a bug in the last PR. If a container or view is not in CDF (or cannot be accessed), we should let the regular post validation fail